File: portfolioblog.client/src/models/blog/blog-playlist/music-playlist.ts
export interface MusicPlaylist {
    id: number; // Maps to 'Id'
    encryptedId?: string; // Maps to 'EncryptedId', optional because it's [NotMapped]
    title: string; // Maps to 'Title'
    playlistBody: string; // Maps to 'PlaylistBody'
    photoPath: string; // Maps to 'PhotoPath'
    rank: number; // Maps to 'Rank'
}

File: PortfolioBlog.Server/Models/Blog/Articles/MusicPlaylist.cs
using PortfolioBlog.Server.Models.Blog.Articles.ArticleSubEnums;
using System.ComponentModel.DataAnnotations.Schema;

namespace PortfolioBlog.Server.Models.Blog.Articles
{
    public class MusicPlaylist
    {
        public int Id { get; set; }
        [NotMapped]
        public string EncryptedId { get; set; }
        public string Title { get; set; }
        public string PlaylistBody { get; set; }
        public string PhotoPath { get; set; }
        public double Rank { get; set; }
    }
}
